class.jetpack.php

```diff
@@ -669,6 +670,15 @@ function point_edit_links_to_calypso( $default_url, $post_id ) {
 		return esc_url( sprintf( 'https://wordpress.com/%s/%s/%d', $path_prefix, $site_slug, $post_id ) );
 	}
 
+	function point_edit_comment_links_to_calypso( $url ) {
+		// Take the `query` key value from the URL, and parse its parts to the $query_args. `amp;c` matches the comment ID.
+		wp_parse_str( wp_parse_url( $url, PHP_URL_QUERY ), $query_args );
+		return esc_url( sprintf( 'https://wordpress.com/comments/all/%s/?commentId=%d&action=edit',
```

Comment:
@kwight the hotlink for edit will be `https://wordpress.com/comment/{site_slug}/{comment_id}?action=edit`

This is Calypso's PR https://github.com/Automattic/wp-calypso/pull/20777